### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -3,16 +3,34 @@
...
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = {};
+  // const properties = [];
+  // const styles = [];
```
Comment (VitaliyBodnar) [inline src/convertToObject.js:19]: remove all comments

File: src/convertToObject.js [hunk 2]
```diff
@@ -3,16 +3,34 @@
...
+  const stylesArr = sourceString.split(';');
+
+  for (const property of stylesArr) {
+    const splittesProperty = property.split(':');
+
+    if (splittesProperty[0].trim().length
+    && splittesProperty[1].trim().length > 0) {
```
Comment (VitaliyBodnar) [inline src/convertToObject.js:26]: it's better to use array destructuring here, for instance:
`const [key, value] = splittesProperty`;

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/convertToObject.js [hunk 3]
```diff
@@ -3,16 +3,32 @@
...
+
+  for (const property of stylesArr) {
+    const splittesProperty = property.split(':');
+
+    const [key, value] = splittesProperty;
+
+    if (key.trim().length
+    && value.trim().length > 0) {
+      resultObj[key.trim()] = value.trim();
+    }
```
Comment (VitaliyBodnar) [inline src/convertToObject.js:28]: let's align your code 
```suggestion
    const splittesProperty = property.split(':');
    const [key, value] = splittesProperty;

    if (
      key.trim().length &&
      value.trim().length
    ) {
      resultObj[key.trim()] = value.trim();
    }
```

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/convertToObject.js (hunk 3 above)
Comment (SerhiiUnhurian) [reply src/convertToObject.js:28]: okay

**Review by Viktor-Kost — APPROVED** [review]

**Pull request closed without merging**